packages/core/src/components/icon/icon.tsx

```diff
@@ -1,21 +1,168 @@
+  { alias: 'chevron-left', svg: chevronLeft },
+  { alias: 'chevron-right', svg: chevronRight },
+  { alias: 'chevron-up', svg: chevronUp },
+  { alias: 'circle-notch', svg: circleNotch },
+  { alias: 'clock', svg: clock },
+  { alias: 'copy', svg: copy },
+  { alias: 'crown', svg: crown },
+  { alias: 'cultural', svg: cultural },
+  { alias: 'download', svg: download },
+  { alias: 'email', svg: email },
+  { alias: 'exclamation', svg: exclamation },
+  { alias: 'external-link', svg: externalLink },
+  { alias: 'eye-slash', svg: eyeSlash },
+  { alias: 'eye', svg: eye },
+  { alias: 'filter', svg: filter },
+  { alias: 'forbidden', svg: forbidden },
+  { alias: 'house', svg: house },
+  { alias: 'info-active', svg: infoActive },
+  { alias: 'info-i', svg: infoI },
+  { alias: 'info', svg: info },
+  { alias: 'infrastructure', svg: infrastructure },
+  { alias: 'label', svg: label },
+  { alias: 'landscape', svg: landscape },
+  { alias: 'location', svg: location },
+  { alias: 'lock', svg: lock },
+  { alias: 'magnet', svg: magnet },
+  { alias: 'map-location', svg: mapLocation },
+  { alias: 'marker', svg: marker },
+  { alias: 'measurement', svg: measurement },
+  { alias: 'minus-square', svg: minusSquare },
+  { alias: 'more', svg: more },
+  { alias: 'municipality', svg: municipality },
+  { alias: 'nature', svg: nature },
+  { alias: 'paperclip', svg: paperclip },
+  { alias: 'parking', svg: parking },
+  { alias: 'pencil', svg: pencil },
+  { alias: 'pin', svg: pin },
+  { alias: 'plus-square', svg: plusSquare },
+  { alias: 'plus', svg: plus },
+  { alias: 'print', svg: print },
+  { alias: 'redo', svg: redo },
+  { alias: 'search', svg: search },
+  { alias: 'sitemap', svg: sitemap },
+  { alias: 'soil', svg: soil },
+  { alias: 'sort-ascending', svg: sortAscending },
+  { alias: 'sort-descending', svg: sortDescending },
+  { alias: 'sort', svg: sort },
+  { alias: 'status-danger', svg: statusDanger },
+  { alias: 'status-info', svg: statusInfo },
+  { alias: 'status-success', svg: statusSuccess },
+  { alias: 'status-warning', svg: statusWarning },
+  { alias: 'table', svg: table },
+  { alias: 'times', svg: times },
+  { alias: 'trash', svg: trash },
+  { alias: 'undo', svg: undo },
+  { alias: 'user-line', svg: userLine },
+  { alias: 'user', svg: user },
+  { alias: 'users', svg: users },
+  { alias: 'water', svg: water },
+  { alias: 'wip-wip', svg: wipWip }
+] as const;
 
 @Component({
   tag: 'dso-icon',
   styleUrl: './icon.scss',
-  assetsDirs: ['icon-assets'],
   shadow: true
 })
 export class Icon {
   @Prop()
   icon!: string;
 
   render() {
-    return (
-      <svg class={clsx('di', this.icon)}>
-        <use href={`${getAssetPath('./icon-assets/dso-icons.svg')}#${this.icon}`} />
-      </svg>
-    );
+    const icon = icons.find(i => i.alias === this.icon);
+    if (!icon) {
+      throw new TypeError(`Unknown svg: ${this.icon}`);
+    }
+
+    return <span class="icon-container" innerHTML={icon.svg} />
```

Comment:
Waarom kan niet de code van `icon.svg` teruggegeven worden? Dat zal toch altijd 1 root hebben, of is een andere reden dat je dit via `innerHTML` moet verpakken? Die `icon-container` staat me zo tegen.

Reply:
De `render()` moet JSX teruggeven. Er is geen mogelijkheid om een string zonder container terug te geven.